client/galaxy/scripts/components/Sharing.vue

```diff
@@ -25,7 +25,7 @@
                         <span id="item-url-text" style="display: none">
                             {{ item_url_parts[0] }}<span id="item-identifier">{{ item_url_parts[1] }}</span>
                         </span>
-                        <a href="#" id="edit-identifier"><img :src="pencil_url"/></a>
+                        <a id="edit-identifier"><img :src="pencil_url" alt="Edit Share Url"/></a>
```

Comment:
```suggestion
                        <a href="javascript:void(0)" id="edit-identifier"><img :src="pencil_url" alt="Edit Share Url"/></a>
```

Reply:
links that don't have href are often styled differently

Reply:
Good Point! Updated.